source/options.rst

```diff
@@ -84,6 +93,80 @@ To view the created configmap, use the following command:
 
    kubectl describe configmaps cluster1-pxc
 
+Use a Secret Object
+-------------------
+
+The Operator can also store configuration options in `Kubernetes Secrets <https://kubernetes.io/docs/concepts/configuration/secret/>`_.
+This can be useful if you need additional protection for some sensitive data.
+
+You should create a Secret object with a specific name, composed of your cluster
+name and a specific suffix:
+
+* ``my-cluster-name-pxc`` for the Percona XtraDB Cluster Pods,
+* ``my-cluster-name-proxysql`` for the ProxySQL Pods,
+* ``my-cluster-name-haproxy`` for the HAProxy Pods,
+  
+.. note:: To find the cluster name, you can use the following command:
+
+   .. code:: bash
+
+      $ kubectl get pxc
+
+Configuration options should be put inside a specific key:
+
+* ``data.pxc`` key for Percona XtraDB Cluster Pods,
```

Comment:
`data:init.cnf` (although in behind I think this one can be called anyhow since mysql will read any named config file from config dir so `my.cnf` should be ok also)
`data:proxysql.cnf`
`data:haproxy-global.cfg`

For example I was using following command to create config maps (so not secret, but directly config map) - I think we have added the similar steps for PSMDB:
```
kubectl create configmap cluster1-pxc --from-file=init.cnf=init.cnf
kubectl create configmap cluster1-haproxy --from-file=haproxy-global.cfg=haproxy-global.cfg
kubectl create configmap cluster1-proxysql --from-file=proxysql.cnf=proxysql.cnf
```